MvvmCross.Tests/MvvmCross.UnitTest/MvvmCross.UnitTest.csproj

```diff
@@ -2,17 +2,18 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <IsPackable>false</IsPackable>
-    <AssemblyName>MvvmCross.Tests</AssemblyName>
-    <RootNamespace>MvvmCross.Tests</RootNamespace>
+    <AssemblyName>MvvmCross.UnitTest</AssemblyName>
```

Comment:
This naming convention is not picked up by the target and prop files.

Reply:
Only 2 or 3 of the old assemblies were picked up either way, because of naming inconsistencies. Will adjust the prop.